mw-webapp/src/component/tag/Tag.module.scss

```diff
@@ -1,65 +1,43 @@
 @import "src/styles/variables";
 
-.tagContainer {
-  position: relative;
-
-  .cross {
-    visibility: hidden;
-  }
-
-  &:hover .cross {
-    visibility: visible;
-  }
-
-  &:focus-visible .cross {
-    visibility: visible;
-  }
-}
+$tagPaddingBlockLarge: 10px;
+$tagPaddingBlockSmall: 3px;
 
 .tag {
   display: flex;
   align-items: center;
   justify-content: center;
-  border: 1px solid var(--primaryStrokeColor);
+  border: $primaryBorder solid var(--primaryStrokeColor);
   border-radius: $tagBorderRadius;
+  background-color: var(--primaryBgColor);
   color: var(--primaryTextColor);
+  gap: $gapSmall;
+  padding-inline: $paddingMedium;
+
+  .text {
+    overflow: hidden;
+    max-width: calc($minWidthBlock - 2 * $paddingMedium - $gapSmall);
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
 }
 
 .primaryTag {
-  overflow: hidden;
-  max-width: $minWidthBlock;
-  height: 40px;
-  background-color: var(--secondaryBgColor);
-  color: var(--primaryTextColor);
   font-size: $fontSizeBig;
-  padding-inline: $paddingMedium;
-  text-overflow: ellipsis;
-  text-wrap: nowrap;
+  padding-block: $tagPaddingBlockLarge;
+
+  button {
```

Comment:
is it possible to use classname (for example, .removeButton)  instead of tag

Reply:
Done